packages/databricks-vscode/scripts/writeJupyterInitFileWithVersion.ts

```diff
@@ -0,0 +1,13 @@
+import {cpSync} from "node:fs";
+import {jupyterInitScriptVersion} from "../package.json";
+import path from "node:path";
+
+const pythonDir = path.resolve(__dirname, "..", "resources", "python");
+cpSync(
+    path.join(pythonDir, "00-databricks-init.py"),
+    path.join(
+        pythonDir,
+        "generated",
+        `00-databricks-init-${jupyterInitScriptVersion}.py`
```

Comment:
let's use the hash of the file content instead. That way we don't need to manually maintain a version number and also can easily change the script during development.